packages/web-app-user-management/src/components/Users/UsersList.vue

```diff
@@ -31,6 +31,7 @@
     <template #avatar="rowData">
```

Comment:
`#avatar="{ item }"` should work too (at least ive seen it like this in other octable implementations)

Reply:
good point :heart_eyes: